helm/kvm-operator-chart/templates/configmap.yaml

```diff
@@ -9,6 +9,9 @@ data:
       listen:
         address: 'http://0.0.0.0:8000'
     service:
+      guest:
+        update:
+          enabled: '{{ .Values.Installation.V1.Guest.Update.Enabled }}'
```

Comment:
We want to configure single installations, e.g. geckon, to activate updates on reconciliation. Goal is to achieve a first MVP where updates functionally work. We want to ignore graceful updates and draining for now and get a first milestone done on this topic. Once we got the updates sorted for customer use we can enable it everywhere. I think this is good since we can switch update support quite easily.